index.html

```diff
@@ -71,7 +71,7 @@ <h2 id="main-h2">Private Digital Currency</h2>
                         </div>
                         <div class="col-lg-6 col-md-6 col-sm-8 col-xs-12 why-text">
                             <h3>Monero is fungible.</h3>
-                            <p> Monero is fungible because it is private by default. In its current state, it is extremely unlikely that Monero will ever be blacklisted by exchanges or vendors due to its association in previous transactions.</p>
+                            <p> Monero is fungible because it is private by default. This means no one can tell one Monero from another Monero and therefore it can't be blacklisted by exchanges or vendors.</p>
```

Comment:
Sorry, if I wasn't clear. Maybe if you clarify that a specific coin can't be blacklisted (because it's funglible)... The wording here isn't clear because it reads as of Monero as a whole can't be blacklisted by an exchange.